README.md

```diff
@@ -35,17 +35,49 @@ The maker and taker frontend depend on the respective daemon running.
 
 We use a separate react projects for hosting taker and maker frontends.
 
-At the moment you will need a browser extension to allow CORS headers like `CORS Everywhere` ([Firefox Extension](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/)) to use the frontends.
+### Building the frontends
 
-### Taker
+The latest version of the built frontends will be embedded by `cargo` inside
+their respective daemons and automatically served when the daemon starts.
+Embedded frontend is served on ports `8000` and `8001` by default.
+
+This means that it is highly recommended to build the frontend *before* the daemons.
+
+#### Taker
+
+```bash
+cd taker-frontend
+yarn install
+yarn build
+```
+
+#### Maker
+
+```bash
+cd maker-frontend
+yarn install
+yarn build
+```
+
+### Developing frontend code
+
+If hot-reloading of the app is required, frontend can be started in development mode.
+Development frontend is served on ports `3000` and `3001` by default.
+
+Note: At the moment you will need a browser extension to allow CORS headers like
+`CORS Everywhere` ([Firefox
+Extension](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/)) to
+use dev frontends.
+
```

Comment:
I don't use this addon 🤷‍♂️

Reply:
removed the addon mention (it was part of the old README)